cdap-ui/package.json

```diff
@@ -33,34 +33,32 @@
     "babel-preset-env": "2.0.0-alpha.20",
     "babel-preset-react": "7.0.0-alpha.20",
     "babel-preset-stage-2": "6.24.1",
-    "bower": "1.7.2",
-    "browser-sync": "2.18.8",
+    "bower": "1.8.2",
     "case-sensitive-paths-webpack-plugin": "2.1.1",
-    "concurrently": "3.1.0",
     "copy-webpack-plugin": "4.0.1",
     "css-loader": "0.28.7",
     "del": "2.2.0",
     "enzyme": "2.6.0",
-    "es6-promise": "3.0.2",
+    "es6-promise": "4.1.1",
```

Comment:
nit: Now that we run on node 8+ do we need this polyfill? If yes can we see if we can try to re-use babel-polyfill? It already has an implementation of promise.